doc/Developer-How-To-Move-Classes-to-src.md

```diff
@@ -0,0 +1,105 @@
+How To Move Classes to `src`
+==============
+
+* [Home](help)
+  * [Developer Intro](help/Developers-Intro)
+
+Since April 2017, Friendica uses [Composer](help/Composer) to manage autoloading.
```

Comment:
I think it would be better to refer to the Friendica version (3.5.2) instead of the date.